exonum-java-binding/packaging/pom.xml

```diff
@@ -5,32 +5,37 @@
   <parent>
     <groupId>com.exonum.binding</groupId>
     <artifactId>exonum-java-binding-parent</artifactId>
-    <version>0.5-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
+    <version>0.6.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>exonum-java-binding-packaging</artifactId>
-  <version>0.5-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <version>0.6.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Exonum Java Binding: Packaging</name>
+  <description>A fake module used for packaging EJB App</description>
 
   <properties>
+    <!--A directory used for storing immediate files while packaging the App-->
     <packaging.basedir>${project.parent.basedir}/core/rust/target/prepackage</packaging.basedir>
+    <!--A directory containing Java dependencies of the App-->
```

Comment:
This comment does not seem correct. Also, I also don't see the property being used. Maybe the other two properties shall use it?

Reply:
🤦‍♂️ I love IDEA revert changes algorithm, which just swapped the lines. This property is used in app_assembly.xml configuration, I can't define properties directly inside it.